cmdline/src/main/java/io/opentdf/platform/Command.java

```diff
@@ -84,10 +93,17 @@ void encrypt(
     }
 
     private SDK buildSDK() {
-        return new SDKBuilder()
-                .platformEndpoint(platformEndpoint)
+        SDKBuilder builder = new SDKBuilder();
+        if (insecure){
+            SSLFactory sslFactory = SSLFactory.builder()
+            .withUnsafeTrustMaterial() // Trust all certificates
+            .build();
+            builder.sslFactory(sslFactory);
+        }
+        
+        return  builder.platformEndpoint(platformEndpoint)
                 .clientSecret(clientId, clientSecret)
-                .useInsecurePlaintextConnection(insecure)
+                .useInsecurePlaintextConnection(plaintext)
```

Comment:
Will this ever conflict with the sslFactory?

Reply:
sslfactory is only used if plaintext isnt set, when plaintext set it just defaults to http and doesnt use the factory
https://github.com/opentdf/java-sdk/blob/7069b1ba5ddc7098a7997cdcfcfdaddb6f899b63/sdk/src/main/java/io/opentdf/platform/sdk/SDKBuilder.java#L227-L237

Reply:
if neither are set it looks like it just does `channelBuilder = ManagedChannelBuilder.forTarget(endpoint);`